.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,60 @@
+name: Weekly maintenance/update tasks
+on:
+    # workflow_dispatch allows for manual triggering of the workflow
+    # Useful for testing that it works without waiting on the cron schedule.
+    workflow_dispatch:
+    schedule:
+        # https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#schedule
+        #        ┌─────────── minute (0 - 59)
+        #        │ ┌───────── hour (0 - 23)
+        #        │ │ ┌─────── day of the month (1 - 31)
+        #        │ │ │ ┌───── month (1 - 12 or JAN-DEC)
+        #        │ │ │ │ ┌─── day of the week (0 - 6 or SUN-SAT)
+        #        v v v v v
+        - cron: 30 7 * * 1
+        #       Run on Mondays @ 7:30am UTC (3:30am EDT, 12:30 am PDT)
```

Comment:
**praise:** Thanks for adding the comments here. I am wondering if it's useful to also note that all times are in UTC. It's tripped me up before and while the docs mention it, it might be useful to add in these comments too. Not super important (something to carry over to other places too), but thought I'd mention the idea.

Reply:
I've noted that the time is in UTC below, but I'll make it explicit.